rules/windows/process_creation/proc_creation_win_office_from_proxy_executing_regsvr32_payload2.yml

```diff
@@ -1,13 +1,17 @@
-title: Excel Proxy Executing Regsvr32 With Payload
+title: Excel Proxy Executing Regsvr32 With Payload Alternate
```

Comment:
This rule and 9d1c72f5-43f0-4da5-9320-648cf2099dd0 should be merged. I'll accept them here, but I will modify them in a PR that I'm gonna push.

Reply:
Need to find a easy way to find overlaps or duplicate rules. But I don't see one for the moment

Reply:
I don't think there is a "best way". We just have to go through them eventually. And make sure that new ones added aren't already covered